projects/dev/src/app/shared/services/api/group.service.ts

```diff
@@ -43,13 +43,17 @@ export class GroupService {
 
   getManagedRequests(
     id,
-    sort = GROUP_REQUESTS_API.defaultSort
+    sort = []
   ): Observable<PendingRequest[]> {
+    let param = {};
+    if (sort.length > 0) {
+      param = {
+        sort: sort.join(",")
+      };
```

Comment:
These 3 lines should be `param.sort = sort.join(",")`, otherwise you would override the previous param you would have given.

Reply:
okay

Reply:
But that statement gives an error actually as param doesn't have sort field.
Instead, I am gonna use this.
```
    let params = new HttpParams();
    if (sort.length > 0) {
      params = params.set('sort', sort.join(","));
    }
```

Reply:
Better 👍